src/monitoring/MonitoringWorker.ts

```diff
@@ -177,6 +179,10 @@ export default class MonitoringWorker {
   }
 
   driver() {
+    const gb = getNewGrowthbookInstance({
+      clientKey: config.get("growthbook.clientKey"),
+      subscribeToChanges: true,
```

Comment:
just a note here, because we set `subscribeToChanges: true`, the same issue where the config changes in-processing still remains. 

this isn't really an issue for me because it only concerns whitelisted sites + whether this is enabled. leave it to you to make the final judgement. how it behaves now is that it initialises 2 new instances and gets the config at that time (whilst listening to changes); if we want to say that the config doesn't change at all in-processing, we could instead initialise growthbook once and get the config at the start and pass it downwards.

tl;dr: behaviour still doesn't change, which means that config can change in-processing. not a biggie for me but leave it to you to decide if you wanna make this change